draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -4096,12 +4098,14 @@ Content-Range: exampleunit 11.2-14.3/25
    provides mechanisms for <x:ref>content negotiation</x:ref>.
 </t>
 <t>
-   This specification defines two patterns of content negotiation that can
+   This specification defines three patterns of content negotiation that can
    be made visible within the protocol:
-   "proactive", where the server selects the representation based
-   upon the user agent's stated preferences, and "reactive" negotiation,
+   "proactive" negotiation, where the server selects the representation based
+   upon the user agent's stated preferences, "reactive" negotiation,
    where the server provides a list of representations for the user agent to
-   choose from. Other patterns of content negotiation include
+   choose from, and "request payload" negotiation, where the user agent
+   selects the representation for a future request based upon the server's
+   stated preferences. Other patterns of content negotiation include
```

Comment:
"stated preferences in past responses." ?

Reply:
done